src/CommonTypes.jl

```diff
@@ -441,6 +441,21 @@ function AutoconversionB1994(
     )
 end
 
+struct AutoconversionVarTimescale{FT}
+    τ::FT
+    α::FT
+end
+
+function AutoconversionVarTimescale(
+    ::Type{FT},
```

Comment:
```
julia> CM.CommonTypes.AutoconversionVarTimescale(Float64, Float64)
ERROR: MethodError: CloudMicrophysics.CommonTypes.AutoconversionVarTimescale(::Type{Float64}, ::Type{Float64}) is ambiguous.

Candidates:
  CloudMicrophysics.CommonTypes.AutoconversionVarTimescale(τ::FT, α::FT) where FT
    @ CloudMicrophysics.CommonTypes ~/clima/CloudMicrophysics.jl/src/CommonTypes.jl:445
  CloudMicrophysics.CommonTypes.AutoconversionVarTimescale(::Type{FT}, toml_dict) where FT
    @ CloudMicrophysics.CommonTypes ~/clima/CloudMicrophysics.jl/src/CommonTypes.jl:449

Possible fix, define
  CloudMicrophysics.CommonTypes.AutoconversionVarTimescale(::Type{FT}, ::Type{FT}) where FT
  ```
  These constructors are ambiguous - the easiest fix is to make `toml_dict` a kwarg.
```suggestion
    ::Type{FT};
```